pset-functions.js

```diff
@@ -33,6 +33,20 @@ console.log('Problem 1:')
 
 // Add your code above this line
 
+function blackJack(playerCardScore, dealerCardScore) {
+    const MAX = 21;
+    if(playerCardScore > MAX && dealerCardScore > MAX){
+        return 0;
+    } else {
+        if(playerCardScore >= playerCardScore && dealerCardScore <= MAX || dealerCardScore > MAX){
```

Comment:
@Ashbk32 the algorithm here is off - the first part of your condition: `playerCardScore >= playerCardScore` will always evaluate to true, which will result in the first value always being returned.  

See the solution for an example of branching logic that will result in the expected result: https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L33